README.md

```diff
@@ -20,9 +20,9 @@ The config passed to `Configure` can contain the following fields.
 | `connection`     | Snowflake connection string.<br/>Supported formats:<br><code>user:password@my_organization-my_account/dbname/schemaname</code> or <br><code>username[:password]@hostname:port/dbname/schemaname </code><br><b>Important</b>: Schema is required | yes      | "user:password@my_organization-my_account/mydb/schema" |
 | `table`          | The table name in snowflake db.                                                                                                                                                                                                                 | yes      | "users"                                                |
 | `columns`        | Comma separated list of column names that should be included in each Record's payload. By default: all columns.                                                                                                                                 | no       | "id,name,age"                                          |
-| `primaryKey`     | Column name that records should use for their `Key` fields.                                                                                                                                                                                     | yes      | "id"                                                   |
-| `orderingColumn` | The name of a column that the connector will use for ordering rows. Its values must be unique and suitable for sorting, otherwise, the snapshot won't work correctly.                                                                           | yes      | "id"                                                   |
-| `snapshot`       |  Whether or not the plugin will take a snapshot of the entire table before starting cdc mode, by default true.                                                                                                                                  | no       | "false"                                                |
+| `primaryKeys`    | Column names that records should use for their `Key` fields.                                                                                                                                                                                    | no       | "id,name"                                              |
```

Comment:
Did you check that snowflake supports composite primary keys?

Reply:
Yes, of course:
https://docs.snowflake.com/en/sql-reference/sql/show-primary-keys.html

> Lists primary keys for the specified table, or for all tables in the current or specified schema, or for all tables in the current or specified database, or for all tables in the current account.